docs/rpc-spec.md

```diff
@@ -604,6 +604,7 @@ except:
 * `rpc-version`
 * `session-id`
 * `version`
+* `units`
```

Comment:
Please insert this alphabetically, e.g. between `session-id` and `version`

Reply:
Done, thanks :+1: